ibis/common/grounds.py

```diff
@@ -219,15 +232,18 @@ def __cached_equals__(self, other):
         return result
 
 
-class Concrete(Immutable, Comparable, Annotable):
+# TODO(kszucs): consider to move it somewhere else, like ibis/expr/core.py
+class Concrete(Immutable, Comparable, Traversable, Annotable):
```

Comment:
It doesn't seem desirable to have every ibis object be traversible. I would expect `Traversable` to exclude data types and schemas and effectively be a filter for `ops.Node`. Why would data types and schemas ever need to be traversable in the same way that `Node`s are?